web/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Context path="/web"/>
```

Comment:
Je to hle opravdu potreba? Nestaci to povoleni statickych souboru v configu?

Reply:
to popravde nevim, mozna by ano